data/credentials/credential-0.json

```diff
@@ -5,6 +5,8 @@
   ],
   "type": ["VerifiableCredential"],
   "issuer": "did:example:123",
-  "issuanceDate": "2021-01-01T19:23:24Z",
-  "credentialSubject": {}
+  "issuanceDate": "2022-03-14T21:40:24Z",
+  "credentialSubject": {
+    "id": "did:example:123"
```

Comment:
Per the spec, `id` is not required... lets add a property that preserves this test case, but that does not leave us with an an empty object.

Reply:
I suggest we add `{ "@vocab": "https://vocab.example#undefined-term" }` to the context, "foo":"bar" to the subject.

Reply:
Amended to use `"foo":"bar"` instead of `id`. Added `{ "@vocab": "https://example.com/#" }` for consistency with the other credentials. Does that work?